kinto_attachment/views.py

```diff
@@ -99,7 +99,7 @@ def delete_attachment_view(request, file_field):
     record["data"][file_field] = None
     utils.patch_record(record, request)
 
-    raise httpexceptions.HTTPNoContent()
+    return {}
```

Comment:
Is this because the exception causes the transaction to abort? I feel like we shouldn't change the API for this -- is there a non-exceptional way to return a 204?